### ISSUE TITLE
[BUG]: SfQuantitySelector not allow to set min === 0

### ISSUE THREAD

**Issue by bartoszherba** [vuestorefront/storefront-ui#2418]
**Describe the bug**
I want to set minimum value to 0 because there are cases for grouped products (Magento 2) when I do not want to add some products to the cart. But I am not able to define min === 0 because of line 92: `const minimum = min || 1;` => 0 || 1 === 1.

**How to reproduce**
The description is self-explanatory.

**Expected behavior**
I am able to set 0 as a min value (but not negative).

**Actual behavior**
I am NOT able to set min value 0 because of the logic issue.

**Code examples**

```
<SfQuantitySelector
  v-model="groupedItem.qty"
  :disabled="loading || !canAddToCart"
  :min="0"
/>
```

**Screenshots**

**Devices (please complete the following information):**

* Device: [e.g. iPhone6, Desktop]
* OS: [e.g. iOS8.1]
* Browser: [e.g chrome, safari][version]

**Solution**
`const minimum = min >= 0 ? min : 1;`

┆Issue is synchronized with this [Jira Zadanie](https://vsf.atlassian.net/browse/SFUI-154) by [Unito](https://www.unito.io)

**Comment by shatanov** [comment]
Hi, I can solve this issue and test the proposed solution)

**Comment by AdamPawlinski** [comment]
@shatanov I can assig it to you, if you are still interested?

**Comment by AdamPawlinski** [comment]
@shatanov Thanks :)

**Comment by FRSgit** [comment]
I'm closing this issue as it was fixed in #2466
